apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/components/template-selector-control.js

```diff
@@ -13,43 +13,56 @@ import { BaseControl } from '@wordpress/components';
 function TemplateSelectorControl( {
 	label,
 	className,
-	selected,
+	useBlank,
 	help,
 	instanceId,
-	onChange,
+	onClick,
 	templates = [],
 } ) {
 	const id = `inspector-radio-control-${ instanceId }`;
-	const onChangeValue = event => onChange( event.target.value );
+	const handleButtonClick = event => onClick( event.target.value );
+
+	if ( isEmpty( templates ) ) {
+		return null;
+	}
+
+	if ( useBlank ) {
+		templates.unshift( {
+			label: 'Blank',
+			preview: 'https://via.placeholder.com/200x180',
+			value: '',
+		} );
```

Comment:
Rather than introducing the `useBlank` prop here, should we add that blank template on the PHP side as a default?